test/functional/kernel/region/test-kernel-region-sync-openmp.cpp

```diff
@@ -0,0 +1,70 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "tests/test-kernel-region-sync.hpp"
+
+#include "../../forall/test-forall-utils.hpp"
+
+using OpenMPKernelRegionSyncExecPols = 
+  camp::list< 
+
+    RAJA::KernelPolicy<
+      RAJA::statement::Region<RAJA::omp_parallel_region,
+
+        RAJA::statement::For<0, RAJA::omp_for_nowait_exec,
+          RAJA::statement::Lambda<0, RAJA::Segs<0>>
+        >,
+
+        RAJA::statement::OmpSyncThreads,
```

Comment:
I _think_ this should work if we remove this sync, because the first 2 for loops are operating on different arrays. The next sync is necessary though.

Reply:
Probably, but it can't hurt to test it.

Reply:
Yes, I suppose the other test-kernel-region-openmp will catch that type of error.